types/shopping-list/ShoppingList.raml

```diff
@@ -55,7 +55,7 @@ properties:
     description: |-
       Human-readable identifiers usually used as deep-link URL to the related shopping list.
       Each slug is unique across a project, but a shopping list can have the same slug for different languages.
-      The slug must match the pattern {{ site.data.api-limits.slugPattern }}.
+      The slug must match the pattern [a-zA-Z0-9_-]{2,256}.
```

Comment:
Just as a remark: This can even expressed on the property:
```yaml
slug?:
  pattern: [a-zA-Z0-9_-]{2,256}
```
But I'm fine with keeping it in the doc for now.
```